src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -122,7 +102,22 @@ export default App.extend({
   onStart(options, collection) {
     this.collection = collection;
     this.filteredCollection = collection.clone();
+    this.editableCollection = collection.clone();
```

Comment:
Nice, this seems like the best way to do it. Just a separate collection like we already do with the `filteredCollection`.

Reply:
I did notice a bug when selecting all via the bulk select button after changing actions to read only: [screengrab](https://github.com/RoundingWell/care-ops-frontend/assets/35355575/d8b659bf-320c-4d2b-9a66-cabf3a6e3038).

The total selected count is off and the newly changed read only action is edited along with the others when it shouldn't.

So I think we need to update the `editableCollection` when an action/flow in the worklist is changed from editable => read only. So any read only items are removed from the `editableCollection`. Currently those read only items are sticking around in there.

Everything else works well with it. Including the `shift`+`click` multi-select.

Reply:
good catch.